coverage_point_calculator/src/lib.rs

```diff
@@ -358,10 +339,6 @@ impl RadioType {
     pub fn is_wifi(&self) -> bool {
         matches!(self, Self::IndoorWifi | Self::OutdoorWifi)
     }
-
```

Comment:
Should we also drop `is_wifi()` above. Could clean up some match statements where it's used.